public/app/features/panel/panel_menu.js

```diff
@@ -12,6 +12,7 @@ function (angular, $, _) {
       var linkTemplate =
           '<span class="panel-title drag-handle pointer">' +
             '<span class="panel-title-text drag-handle">{{panel.title | interpolateTemplateVars:this}}</span>' +
+            '<span class="panel-tooltip"><dtip>panel.helpText</dtip></span>' +
             '<span class="panel-links-btn"><i class="fa fa-external-link"></i></span>' +
```

Comment:
@torkelo At this line if 
```
<tip>{{panel.helpText}}</tip>
```
would have worked, then we do not need **dtip** directive. But **tip** directive is not able to resolve this variable. :(